src/main/java/sirius/biz/web/ComplexLoadProperty.java

```diff
@@ -28,4 +28,17 @@
      * @return <tt>true</tt> if loading succeeded, <tt>false</tt> if an error occurred
      */
     boolean loadFromWebContext(WebContext webContext, BaseEntity<?> entity);
+
+    /**
+     * Determines if "autoloading" should be used for the given request.
+     * <p>
+     * This can inspect the request a check if the required parameters are present in order to successfully perform
+     * an automatic load.
+     * <p>
+     * Note that a default property would check for {@code webContext.hasParameter(this.getPropertyName())}.
+     *
+     * @param webContext the request to inspect
+     * @return <tt>true</tt> if the proper parameters are filled so that the property can be loaded
+     */
+    boolean shouldAutoload(WebContext webContext);
```

Comment:
maybe we could rename this - i think its kind of confusing that it has the same name as the method in the biz controller and from a method called "shouldAutoload" i would expect it to also check the annotation/permission.

I don't have a good suggestion tho, my only is maybe "hasLoadableParameter"

Reply:
the BizController.shouldAutoload calls this method:
```
        if (property instanceof ComplexLoadProperty complexProperty) {
            return complexProperty.shouldAutoload(webContext);
        }
```